stories/Popover/Popover.stories.jsx

```diff
@@ -0,0 +1,201 @@
+import Popover from '../../components/Popover';
+import { Row, Col } from '../../components';
+
+const Template = args => {
+  const { children: Component, ...rest } = args;
+  return (
+    <>
+      <Row style={{ paddingTop: '72px', paddingLeft: '414px' }}>
```

Comment:
We can follow the stylization adopted in the tooltip documentation in this case of not centering the popover using paddingLeft